nilearn/tests/test_signal.py

```diff
@@ -274,6 +287,36 @@ def test_clean_detrending():
     assert_false(abs(x_undetrended - signals).max() < 0.06)
 
 
+def test_clean_t_r():
+    """Different TRs must produce different results after filtering"""
+    n_samples = 34
+    # n_features  Must be higher than 500
+    n_features = 501
+    x_orig = generate_signals_plus_trends(n_features=n_features,
+                                          n_samples=n_samples)
+    random_tr_list1 = np.round(np.random.rand(5) * 10, decimals=2)
+    random_tr_list2 = np.round(np.random.rand(5) * 10, decimals=2)
```

Comment:
There is an unseeded random number generator here. We cannot do that: every random number generator in the tests must be seeded.

Reply:
Also, that list is quite long: given that we test with two values for low and high pass filters below, do we need to draw 5 values? Maybe 3 is enough?

Reply:
Thanks.. Good points. I pushed a change. Please have a look.